src/main/java/com/example/bankservice/service/UserService.java

```diff
@@ -0,0 +1,13 @@
+package com.example.bankservice.service;
+
+import com.example.bankservice.model.User;
+
+public interface UserService {
+    void add(User user);
+
+    User getById(Long id);
+
+    User getByPhoneNumber(String phoneNumber);
+
+    void remove(Long id);
+}
```

Comment:
You also need to declare method `update`

Reply:
Not really, it`s from the checklist "You can have a single save() method on the service layer for both ‘create’ and ‘update’ operations."
So maybe i`ll just rename add() to save()

Reply:
Ok, it's optional but in my opinion, you can declare `add` and `update`. And they both will use `save` from repository

Reply:
@vicire I agree with Lena on this point - there is no particular reason to create 2 method with the same realisation